packages/mui-codemod/README.md

```diff
@@ -1179,6 +1179,32 @@ CSS transforms:
 npx @mui/codemod@next deprecations/step-connector-classes <path>
 ```
 
+#### `list-item-props`
```

Comment:
May I ask you to move this section so it's ordered alphabetically? 😊

Reply:
Done! I took the opportunity to sort out some other sections that were out of place.